Xwt.Mac/NSStringAttributeKey.cs

```diff
@@ -0,0 +1,43 @@
+﻿using MonoMac.Foundation;
+
+namespace Xwt.Mac
+{
+	public static class NSStringAttributeKey
+	{
+		public static NSString ForegroundColor
+		{
+			get { return NSAttributedString.ForegroundColorAttributeName; }
```

Comment:
This will definitely break Xamarin.Mac, since this was removed.

We need to introduce some define constants and most likely a new configuration to handle MonoMac and provide extension/wrapper API inside Xwt that provides compatibility.